@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -481,6 +481,10 @@ export interface TerrainTechnique extends MakeTechniqueAttrs<TerrainTechniquePar
     name: "terrain";
 }
 
+export interface LabelRejectionLineTechnique extends MakeTechniqueAttrs<BaseTechniqueParams> {
+    name: "label-rejection-line";
```

Comment:
@ninok , do you have any preference?

Reply:
"label-separation-line" sounds a bit strange in my opinion, as i would question what it is separating. "Label-Reservation" together sounds a bit weird, it is a line that is reserving for labels, does that mean it reserves some space to make Labels visible?

Reply:
I was just a bit worried that the "rejection" could be understood as a line that keeps label from appearing, but I guess in the final implementation the labels will be moved to a different location, no? Alternatives (dict.cc is my friend...):

label-displacement-line
label-lock-out-line
label-relocation-line
label-push-back-line

... let the native speaker(s) decide!

Reply:
> I was just a bit worried that the "rejection" could be understood as a line that keeps label from appearing,

Good, because that is exactly what it does, it prevents labels from appearing.